tianshou/utils/net/continuous.py

```diff
@@ -137,6 +143,14 @@ def __init__(
             flatten_input=flatten_input,
         )
 
+    def __setstate__(self, state: dict) -> None:
```

Comment:
I don't mind doing that (although I doubt anyone actually persistent this type of net since 1.0 release), but let us remember to remove it at release of 2.0. Could you add a TODO-2.0: remove, so we can find it later?

Also, pls add a description of the fix and api extension to `CHANGELOG.md`

Reply:
Why do you doubt that? It's quite likely actually, as it would affect _everyone_ who persisted a policy that contains a continuous.Critic. Persisting the policy is the default behaviour of the high-level API, so it would be unlikely only if you assume that we have no users. ;)

And the TODO does not make sense at this stage, because this class has not yet been changed (incompatibly) for 2.0. Once it is changed for 2.0, the one changing it should add the TODO - not me and not now.

Reply:
Ok :)

I'll merge this without squashing in a few hours. If you wanna clean up the history and force push, now's the chance ;)

Reply:
> Also, pls add a description of the fix and api extension to CHANGELOG.md

We should add this to the PR template.

Reply:
Yes, was on my list but I forgot. Feel free to do that if you want, otherwise I'll push the extended template to master

Reply:
I added it to the template.